### PR TITLE
add interaction rate TOF qc

### PR DESCRIPTION
**Pull request by noferini**
@njacazio @ercolessi 
Added one 2D histos to identify collisions BC

### CODE REVIEW

**Review by njacazio — APPROVED** [review]
Thanks Francesco

**Comment by noferini** [comment]
O2 test failing since token is missing

macOS failing with
49 warnings generated.
ld: library not found for -lcrypto.44
clang: error: linker command failed with exit code 1 (use -v to see invocation)

I would say the PR is clean

**Comment by njacazio** [comment]
Yes, I agree, @Barthelemy @knopers8 could this be merged? Thanks!!

**Comment by noferini** [comment]
I put this WIP, waiting for the trending part developed by Francesca (almost ready)

**Comment by noferini** [comment]
Ready 

This is an example of the outputs

https://qcg-test.cern.ch/?page=objectView&objectName=qc/TOF/MO/TOFTrendingRate/active_channels
https://qcg-test.cern.ch/?page=objectView&objectName=qc/TOF/MO/TOFTrendingRate/noise_rate
https://qcg-test.cern.ch/?page=objectView&objectName=qc/TOF/MO/TOFTrendingRate/pile_up
https://qcg-test.cern.ch/?page=objectView&objectName=qc/TOF/MO/TOFTrendingRate/interaction_rate

File: Modules/TOF/src/TrendingRate.cxx [hunk 1]
```diff
@@ -72,6 +72,10 @@ void TrendingRate::computeTOFRates(TH2F* h, TProfile* hp, std::vector<int>& bcIn
     }
   }
 
+  if(mActiveChannels){
```
Comment (njacazio) [inline Modules/TOF/src/TrendingRate.cxx:75]: This will probably require clang format

**Review by knopers8 — APPROVED** [review]

**Pull request merged**